go/chat/convsource.go

```diff
@@ -121,7 +126,17 @@ func (s *HybridConversationSource) Push(ctx context.Context, convID chat1.Conver
 	var err error
 	continuousUpdate := false
 
-	decmsg, err := s.boxer.UnboxMessage(ctx, msg)
+	// is it possible that this will get called after a conversation has
+	// been finalized?  if so, we need this:
+	// I'm fine with having an error in that case (and thus speeding up this codepath
+	// by avoiding rpc call).
+	var rl []*chat1.RateLimit
+	conv, err := ConversationMetadata(ctx, s.ri(), convID, &rl)
+	if err != nil {
+		return chat1.MessageUnboxed{}, continuousUpdate, err
+	}
```

Comment:
Yeah maybe we should skip this. It will just mean that an incoming Gregor message might show up as an error in a rare care, but in the common case we will be running this query for every send.